elements/nvidia-cuda/pre-install.d/01-nvidia-cuda-repo

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -lv
+
+case "$DISTRO_NAME" in
+    centos* | rhel*)
+	yum install -y https://developer.download.nvidia.com/compute/cuda/repos/rhel7/x86_64/cuda-repo-rhel7-8.0.61-1.x86_64.rpm
```

Comment:
If we're going to assume an OS version, we could verify that DIB_RELEASE is as expected.